server.js

```diff
@@ -1,12 +1,16 @@
 const express = require("express");
 const app = express();
 const userRouter = require("./routes/user");
-const methodLogger = require("./logger/logger");
+const methodLogger = require("./middleware/logger");
 const connectDB = require("./config");
 
+const jwt = require("jsonwebtoken");
+const authenticateRouter = require("./routes/authenticationRoute");
+
 app.use(express.json());
 app.use(methodLogger);
-app.use("/", userRouter);
+app.use("/login", authenticateRouter); // Use authenticateRouter for login route
```

Comment:
I think this route is misleading as refresh and login will have same route /login and is differentiated only by GET or POST

Reply:
will route for refresh be /login/refreshToken? Should it be like this

Reply:
no ram refresh token should be generated by hitting a get request with /refreshToken route